### ISSUE TITLE
Viewing extension popup in external monitor misaligns mouse position

### ISSUE THREAD

**Issue by Yoshi8765** [david-sabata/web-scrobbler#1292]
### Current Behavior
When I view chrome on an external monitor and use the web-scrobbler extension popup, it misunderstands where the mouse is located, for some reason.
For example: http://i.imgur.com/zwDrOBI.png
This makes it very hard to click the heart for the now playing track. If it helps, I am using soundcloud.
I believe this is a bug for the extension because everything else in chrome on the external monitor operates fine, as far as I know.

### Environment

* Version used: 1.63.0
* Browser Name and version: Chrome Version 57.0.2987.133 (64-bit)

**Comment by myfonj** [comment]
Confirming.
Actually this happens to every single extension popup when Chrome window is dragged to secondary monitor with different DPI.
(Made me finally report that: https://bugs.chromium.org/p/chromium/issues/detail?id=715552 )

**Comment by alexesprit** [comment]
I also reproduced this issue on Chrome 58.0.3029.81 and 60.0.3081.0 canary.
I didn't reproducet it on Opera 44.

Looks like it's on Chrome side. Let's wait for Chrome developers comments.

**Comment by Yoshi8765** [comment]
You're right, my apologies. It's an issue with every extension in chrome. Looking around at some google support forum posts, it appears Google's stance is that extension owners should "figure it out," which sounds ridiculous to me... I've tried to set high DPI scaling override to be performed by the System, but this made chrome unviewable on my laptop screen, although it seems to have been fixed on my external monitor.
